wasm.md

```diff
@@ -276,38 +280,44 @@ Comparisons consume two operands and produce a bool, which is an `i32` value.
 
 ```k
     syntax PlainInstr ::= IValType "." IRelOp
- //                     | FValType "." FRelOp
+                        | FValType "." FRelOp
  // -----------------------------------------
 
     syntax Instr ::= IValType "." IRelOp Int   Int
- //                | FValType "." FRelOp Float Float
+                   | FValType "." FRelOp Float Float
  // ------------------------------------------------
     rule <k> ITYPE . ROP:IRelOp => ITYPE . ROP C1 C2 ... </k>
-         <valstack> < ITYPE > C2 : < ITYPE > C1 : VALSTACK => VALSTACK  </valstack>
+         <valstack> < ITYPE > C2 : < ITYPE > C1 : VALSTACK => VALSTACK </valstack>
+    rule <k> FTYPE . ROP:FRelOp => FTYPE . ROP C1 C2 ... </k>
+         <valstack> < FTYPE > C2 : < FTYPE > C1 : VALSTACK => VALSTACK </valstack>
+
 ```
 
 ### Conversion Operations
 
 Conversion operators always take a single argument as input and cast it to another type.
-For each element added to `ConvOp`, functions `#convSourceType` and `#convOp` must be defined over it.
+For each element added to `CvtOp`, functions `#convSourceType` and `#CvtOp` must be defined over it.
```

Comment:
Any special reason to change the name in this PR? Just wondering, it seems unrelated to the rest of the fixes.

Reply:
Well, we can have a separate PR for this. The only reason for this renaming is that the Op is named `CvtOp` in the specs